libs/blocks/modal/modal.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable import/no-cycle */
 import { createTag, getMetadata, localizeLink, loadStyle, getConfig } from '../../utils/utils.js';
+import { getFederatedUrl } from '../global-navigation/utilities/utilities.js';
```

Comment:
We might want to think about exporting the `getFederatedUrl` method to a more generic utilities and decouple it from the Global Navigation.

Reply:
Is it okay if i move this out after this [PR](https://github.com/adobecom/milo/pull/2523) to avoid conflict, as it also touches the utils.js files?

Reply:
Thanks, @bandana147! Is there a JIRA ticket already in your backlog?

Reply:
Created one - https://jira.corp.adobe.com/browse/MWPW-153670